### DISCUSSION TITLE
Favorites List

### DISCUSSION THREAD

**Discussion by noxamos** [gotson/komga#1559] [Ideas]
### Describe your suggested feature

The ability to mark certain series as favorite so they show up in a new page on the sidebar under Home and above Libraries.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by noxamos** [comment]
Related to https://github.com/gotson/komga/issues/900

**Comment by jvlk123** [comment]
+1
